src/utilities/util_verify.c

```diff
@@ -113,25 +119,31 @@ util_verify(WT_SESSION *session, int argc, char *argv[])
             return (usage());
         }
 
+    if (dump_all_data && dump_key_data)
+        WT_RET_MSG((WT_SESSION_IMPL *)session, ENOTSUP, "%s",
+          "dump_all_data, which unredacts all data, should not be set to true "
```

Comment:
we should mention the options -u and -k in this message.

"-u (dump_all_data), which unredacts all data, should not be set with -k (dump_key_data) "